Dockerfile.12sp0

```diff
@@ -17,7 +17,7 @@ RUN wget http://username:password@gaffer.suse.de:9999/files/.regcode -O ~/.regco
 RUN wget http://username:password@gaffer.suse.de:9999/files/.oscrc -O ~/.oscrc
 
 RUN echo 'gem: --no-ri --no-rdoc' > /etc/gemrc && \
-    gem install bundler --no-document
+    gem install bundler -v 1.17.3 --no-document
```

Comment:
I think you can do something like `--version '~> 1.17'` so it would still do minor and patch updates, but not pick up version 2.